BunnyHopMaster/Assets/Scripts/Level/Level.cs

```diff
@@ -7,17 +7,29 @@ public class Level : ScriptableObject
     [Header("Set in Editor")]
     [SerializeField]
     public int levelBuildIndex;
+
     [SerializeField]
     public string levelName;
+
     [SerializeField]
     public bool isPortalLevel;
+
     [SerializeField]
     public string[] tutorialTexts;
+
     [SerializeField]
     public int numberOfCheckpoints;
+
     [Header("Set in Game")]
     [SerializeField]
     public bool isCompleted;
+
     [SerializeField]
     public float completionTime;
+
+    private void OnEnable()
```

Comment:
im not sure this makes sense to have on a scriptable object. can you explain this decision?

Reply:
Well was looking for a way to reset values on build and play and found that this could work